ogre_mesh_viewer.py

```diff
@@ -555,8 +555,8 @@ def setup(self):
         # for picking
         self.ray_query = scn_mgr.createRayQuery(Ogre.Ray())
 
-        imgui_overlay.addFont("SdkTrays/Value", RGN_MESHVIEWER)
-        self.logwin.font = ImGui.GetIO().Fonts.AddFontDefault()
+        imgui_overlay.addFont("Liberation/Sans", Ogre.RGN_DEFAULT)
+        self.logwin.font = imgui_overlay.addFont("Liberation/Mono", Ogre.RGN_DEFAULT)
         self.logwin.font.Scale = round(pixel_ratio)
```

Comment:
this is no longer needed for ttf fonts

Reply:
I don't understand. What is no longer needed?

Reply:
Scale = round(pixel_ratio)